integrations/oci/sdk/runtime/src/main/java/module-info.java

```diff
@@ -16,7 +16,10 @@
 
 /**
  * Helidon Injection Integrations to support OCI Runtime module.
+ *
+ * @deprecated replaced with {@code helidon-integrations-oci} module
  */
+@Deprecated(forRemoval = true, since = "4.1.0")
```

Comment:
Not all classes in this module are marked deprecated e.g. 
OciConfigBlueprint.java, OciRegionProvider.java. Do we need to?

Reply:
Neither of them are public.

Reply:
I will take a note to generate the `OciConfig` with deprecated annotation, and do it as a follow up

Reply:
Created #8806